docs/user/user/adding-glean-to-your-project/javascript.md

```diff
@@ -8,15 +8,19 @@ Currently, these bindings support collecting data from [Browser Extensions](http
 
 ## Requirements
 
-* Node.js >= 12.20.0.
-* npm >= 7.0.0.
-* Webpack >= 5.34.0.
-* Python >= 3.6.[^1]
-* (_for browsers other than Firefox_) [webextension-polyfill](https://github.com/mozilla/webextension-polyfill) >= 0.8.0.[^2]
+* Node.js >= 12.20.0;
+* npm >= 7.0.0;
+* Webpack >= 5.34.0;
+* Python >= 3.6.
+  * The `glean` command requires Python to download [`glean_parser`](https://mozilla.github.io/glean_parser/) which is a Python library.
 
-[^1]: The `glean` command requires Python to download [`glean_parser`](https://mozilla.github.io/glean_parser/) which is a Python library.
+### Web extension specific requirements
 
-[^2]: Glean.js assumes a Promise-based `browser` API: Firefox provides such an API by default. Other browsers may require using a polyfill library such us `webextension-polyfill` when using Glean in browser extensions.
+* ["storage"](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/permissions#api_permissions) API permission;
+* [Host permission](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/permissions#host_permissions) to the Telemetry server;
+  * Only necessary if the defined server endpoint denies [cross-origin](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS) requests.
+* (_for browsers other than Firefox_) [webextension-polyfill](https://github.com/mozilla/webextension-polyfill) >= 0.8.0.[^1]
```

Comment:
This list has several different things. Please call out that this is another npm dependency for the project

Reply:
I tried to add some txt to clarify this, but anything I wrote sounded confusing. It's not really a npm dependency, you can just download the script and load it. The important part is that you load it... Anyways, I added a link to the sample configuration, that is probably much more enlightening than anything I write.

Reply:
ah well, yeah, some sort of source dependency for users.